generators/app/templates/resources/lib/plugin.py

```diff
@@ -13,16 +14,21 @@
 kodiLogging.config()
 plugin = routing.Plugin()
 
+
 @plugin.route('/')
 def index():
-    addDirectoryItem(plugin.handle, plugin.url_for(show_category, "one"), ListItem("Category One"), True)
-    addDirectoryItem(plugin.handle, plugin.url_for(show_category, "two"), ListItem("Category Two"), True)
+    addDirectoryItem(plugin.handle, plugin.url_for(
+        show_category, "one"), ListItem("Category One"), True)
```

Comment:
In my opinion (not that it matters much :) ), the fact that we instantiate and pass `Listitem` objects directly to the addDirectoryItem makes it really simple for new users to understand how to create a new directory item but hides a lot of logic that might be applied to listitems. Having this like this, will probably make users to use `iconImage` and `thumbnailImage` inline with the object instantiation. Both are deprecated and we making an effort for them to use `setArt` instead. Also, despite there are no actual benchmarks, some team members state that addDirectoryItems is much faster than addDirectoryItem when the amount of listitems to add is considerable. So probably a good idea to individually create tem and just call addDirectoryItems with a list (or generator if supported)